docs/README.md

```diff
@@ -1,23 +1,25 @@
 # AnyCable
 
-> Build lightning fast, reliable real-time applications with Ruby and Rails
+> A real-time server for Rails, Node.js, and Hotwire applications that runs on your servers and scales.
 
 <picture>
-     <source srcset="/assets/images/logo_invert.svg" media="(prefers-color-scheme: dark)">
-     <img class="home-logo" align="right" height="150" width="129" title="AnyCable logo" src="/assets/images/logo.svg">
+  <source srcset="/assets/images/logo_invert.svg" media="(prefers-color-scheme: dark)">
+  <img class="home-logo" align="right" height="150" width="129" title="AnyCable logo" src="/assets/images/logo.svg">
 </picture>
 
-AnyCable helps you build reliable and fast real-time features—notifications, chats, Hotwire frontends, and more—natively on Ruby on Rails without using 3rd-party services. Stay productive by writing clean, maintainable code in Ruby with the assurance that your application scales efficiently.
+AnyCable helps you build reliable and fast real-time features—notifications, chats, Hotwire frontends, and more. It works with any backend and provides seamless integrations for Ruby on Rails and serverless Node.js applications. Stay productive by writing clean, maintainable code in your language of choice with the assurance that your application scales efficiently.
 
-Save up to 10x on infrastructure compared to built-in Action Cable without sacrificing Rails productivity by switching to third-party services: a fast Golang server handles the load, a gRPC-driven Rails application deals with business-logic. The [Pro version](./pro.md) offers even more benefits: an additional 40% reduced memory footprint, DX improvements, and priority support.
+Save up on infrastructure and PaaS bills without sacrificing productivity: a fast Go server handles the load, and your application deals with business logic. The [Pro version](./pro.md) offers even more benefits: an additional 40% reduced memory footprint, DX improvements, and priority support.
```

Comment:
Let's update the Pro benefits:
The [Pro version](./pro.md) offers even more benefits: delivery guarantees in cluster mode, fallback transport for private networks, GraphQL integration, added memory efficiency and more.